lib/src/native/realm_core.dart

```diff
@@ -871,7 +880,13 @@ extension on Pointer<realm_value_t> {
 
         return ObjectId.fromBytes(bytes);
       case realm_value_type.RLM_TYPE_UUID:
-        throw Exception("Not implemented");
+        final nativeBytes = ref.values.uuid.bytes;
+        final bytes = List<int>.filled(16, 0);
+        for (int i = 0; i < 16; i++) {
+          bytes[i] = nativeBytes[i];
+        }
+
+        return UuidValue.fromList(bytes);
```

Comment:
```suggestion
        return UuidValue.fromList(cast<Uint8>().asTypedList(16));
```

Reply:
Should be enough, given memory layout.. but the `UuidValue.fromList` ctor is gross